pkg/controller/vsphere/actuator_test.go

```diff
@@ -24,8 +24,15 @@ import (
 
 func init() {
 	// Add types to scheme
-	machinev1.AddToScheme(scheme.Scheme)
-	configv1.AddToScheme(scheme.Scheme)
+	err := machinev1.AddToScheme(scheme.Scheme)
+	if err != nil {
+		panic(err)
+	}
+
+	err = configv1.AddToScheme(scheme.Scheme)
+	if err != nil {
```

Comment:
There's a linter that checks for inlining these, lets enable it
```suggestion
	
	if err := configv1.AddToScheme(scheme.Scheme); err != nil {
```